bin/fterm

```diff
@@ -0,0 +1,323 @@
+      echo "      were reinstalled. Even though these images can still be used it is "
+      echo "      recommended to simply delete them."
+      echo "Images are: "
+      docker images | grep "^<none>"
+
+      if [[ $YES_TO_ALL -eq 1 ]]; then
+          docker rmi -f $UNNAMED
+      else
+          read -p "Would you like to remove them? " -n 1 -r
+          echo
+          if [[ $REPLY =~ ^[Yy]$ ]]
+          then
+              docker rmi -f $UNNAMED
+          fi
+      fi
+      echo "--------------------------------------------"
+  fi
+}
+
+
+# prints usage
+function usage() {
+  echo "usage [options] [dbg|rel] [action] [arguments]"
+  echo "  options:"
+  echo "    -v|--verbose"
+  echo "    |-d|--debug    - turn on debug mode (set -x)"
+  echo ""
+  echo "  actions:"
+  echo "    clean          - will remove all personalized images, the other images"
+  echo "                     will be left untouched"
+  echo ""
+  echo "    remove-old     - will prompt to remove old docker images and containers"
+  echo "                     (those with no name)"
+  echo ""
+  echo "    build          - will create customized images (ending with user)"
+  echo ""
+  echo "    update         - will update all flow123d images (will download new images"
+  echo "                     from the hub.docker.com/u/flow123d if necessary)"
+  echo ""
+  echo "    run            - (default action) will start a personalized image"
+  echo "                     directory specified in config/docker.cfg file"
+  echo "                     currently directory '$work' will be mounted"
+  echo ""
+  echo "    flow123d       - will run flow123d binary and pass all given arguments"
+  echo "                     specified after this value"
+  echo ""
+  echo "    runtest        - will run runtest binary and pass all given arguments"
+  echo "                     specified after this value"
+  echo ""
+  echo "  arguments: additional arguments passed to binary"
+  echo "             (if action is flow123d or run or runtest)"
+  echo ""
+  echo "  To compile flow under docker, you can do the following:"
+  echo "    $REL_BIN/fterm run make -j4 all"
+  echo ""
+  echo "  To run flow under docker, you can do the following:"
+  echo "    $REL_BIN/fterm flow123d --help"
+  echo ""
+  echo "  To run tests under docker, you can do the following:"
+  echo "    $REL_BIN/fterm runtest tests/10_darcy"
+    
+}
+
+
+# get CWD as realtive path to current directory, which enables to build image on Windows
+OLD_PWD="$(pwd)"
+ABS_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )"/.. && pwd )"
+ABS_BIN="$ABS_ROOT/bin"
+ROOT=$(realpath --relative-to=$(pwd) $ABS_ROOT)
+REL_BIN=$(realpath --relative-to=$(pwd) $ROOT/bin)
+cd $ROOT
+REl_ROOL='.'
+
+# define paths
+BIN=$REl_ROOL/bin
+CFG=$REl_ROOL/config
+TMP=$REl_ROOL/config/docker/.tmp
+DKR=$REl_ROOL/config/docker
+
+
+gid=$(id -g)
+uid=$(id -u)
+uname=$(whoami)
+work=${WORK:-$HOME/workspace}
```

Comment:
Use prefix FLOW123D_WORK
If not found will use first lower directory from $HOME, which contains flow123d related to current dir
Given flow123d repo's location: `/home/name/projects/Flow123d/flow123d`
Will mount `/home/name/projects/`